src/templates/installing/os/ubuntu/nginx/install.md

```diff
@@ -1,7 +1,3 @@
-The following is an abbreviation of the [Ubuntu PPA nginx installation instructions](https://www.nginx.com/resources/wiki/start/topics/tutorials/install/#ubuntu-ppa).
```

Comment:
Why remove this? Should people not be installing the latest stable nginx?

It's probably better to use the ppa since people could be on older versions of Ubuntu.

Reply:
See my initial comment on the PR.